services/globoplay.yml

```diff
@@ -0,0 +1,8 @@
7-1.1-7.5-1.1-11.2V18.5c0-3.2.3-6.4.6-9.6.5-4.5 2.6-6.8 7-7.6 2.5-.5 5.1-.6 7.6-.9l3-.5 6.3.1zM818 200.2v51.3l-.1 12a29 29 0 0 1-.8 5c-1.2 4.4-3.9 6.7-8.4 6.9-6.9.3-13.7.5-20.6.5-2.1 0-4.3-.4-6.4-.9a8 8 0 0 1-6-5.6c-.7-2.1-1-4.3-1.3-6.5-.2-1.7-.1-3.4-.1-5V78.2c0-2.4.1-4.8.7-7.1 1-4 3.5-6.4 7.8-7.1 6.8-1 13.5-1.4 20.3-.4 5.2.7 8.6 3.6 10.2 8.6.7 2.2.9 4.5 1.3 6.8l.1 1.2c.3 1.5 1.5 1.9 2.6.9l1.8-1.9a70.8 70.8 0 0 1 31.6-17.3c7.7-1.7 15.5-2 23.4-1.2a61.7 61.7 0 0 1 52.4 35.6c5 10.5 8.1 21.5 9.3 33.1a94 94 0 0 1-9.6 52.6 61.5 61.5 0 0 1-45.3 34.3 66.4 66.4 0 0 1-47-6.2 86 86 0 0 1-13-8.6l-2.1-1.4c-.4-.1-.6 0-.8.1zm-.3-63.9c.1 6.3.9 12.3 2.6 18.3a32.6 32.6 0 0 0 12.3 18.6 37.8 37.8 0 0 0 29.1 6.6 33 33 0 0 0 26.2-21.5 51.5 51.5 0 0 0-2.8-45.1 27.4 27.4 0 0 0-16.3-13.7 41.4 41.4 0 0 0-21.8-1.3c-14.3 3-23.3 12-27.4 25.9-1 3.9-1.3 8.1-1.9 12.2zM323.4 60c14.3-.2 26.9 2 38.9 7.5A69.7 69.7 0 0 1 403 118a89.7 89.7 0 0 1 2.4 20 79.2 79.2 0 0 1-17.9 51.7 72 72 0 0 1-41.5 25.8 96 96 0 0 1-30.2 2.3 83.3 83.3 0 0 1-35.2-11.6 68.5 68.5 0 0 1-32.9-44.4 93.2 93.2 0 0 1-2.8-31.1 77.3 77.3 0 0 1 9.2-30.3A73.7 73.7 0 0 1 305.6 62a68.4 68.4 0 0 1 17.8-2zm-35.1 74.7c0 13.7 2.6 22.8 7.5 30.3 3.3 5 7.6 8.8 13 11.4a39.8 39.8 0 0 0 21.2 3.8c8.3-.9 15.3-4.2 20.8-10.6a39 39 0 0 0 7.7-13.4 56 56 0 0 0-.1-36 35.2 35.2 0 0 0-63.7-5.5 42.8 42.8 0 0 0-6.4 20zm469.9 3.2c0 21.5-7 38.9-20.1 54.1a72 72 0 0 1-39.8 23.4 95 95 0 0 1-29.1 2.3 83 83 0 0 1-42-15.4 69 69 0 0 1-28.1-43.9 95 95 0 0 1-1.9-28.1A77 77 0 0 1 658.7 62a93.8 93.8 0 0 1 31.7-1 78.6 78.6 0 0 1 36.2 13.5 71.3 71.3 0 0 1 29.3 44.9c1.5 6.6 2.1 13.3 2.3 18.5zm-44.3.8c.1-7-1.3-13.8-3.9-20.4a30 30 0 0 0-17-17.9 39 39 0 0 0-25.2-1.7 34 34 0 0 0-24.1 22.5 51.6 51.6 0 0 0 0 35.7 33.6 33.6 0 0 0 19.9 20.8 38.6 38.6 0 0 0 21 2 30.8 30.8 0 0 0 24-18.2 51.5 51.5 0 0 0 5.3-22.8zm406.2 57.4c-1.8.1-2.4 1.2-3.3 2.1a66.7 66.7 0 0 1-19.2 13.7c-6.3 2.9-13 4.5-19.8 5.4-9.3 1.2-18.5 1-27.6-1.4a56.4 56.4 0 0 1-18.8-8.2 40.8 40.8 0 0 1-17.6-30.1c-.7-6.3-.4-12.5 1.2-18.6a39.9 39.9 0 0 1 18-24.3c7.7-5 16.2-7.9 25.2-9.5 10.3-1.8 20.7-1.7 31.1-.6 8 .8 16 2.4 23.8 4.1 2.5.6 3 .2 3-2.4 0-4.4-.7-8.6-2.1-12.7a25.5 25.5 0 0 0-20.9-17.7 68.2 68.2 0 0 0-43.7 5.9c-3 1.5-6.2 2.6-9.6 2.8-4 .3-7.3-.9-9.6-4.2l-5.8-9.1-.4-.9c-3-6.2-1.9-10.7 3.8-14.5a103.5 103.5 0 0 1 57.8-16.9c12.8.1 25.1 2.8 36.8 7.7a54.6 54.6 0 0 1 32.4 37.4 79 79 0 0 1 2.8 21.6l-.2 74.6c0 1.9-.2 3.8-.6 5.7a8.4 8.4 0 0 1-6.2 6.7 48.6 48.6 0 0 1-18.8 1c-4-.5-7-2.7-8.3-6.7-1-2.9-1.7-5.8-2.6-8.7-.1-.6-.5-1.4-.8-2.2zm-35.3-45.1c-2.5.2-5.1.3-7.6.7a56 56 0 0 0-10 2.4c-6 2.1-9.9 6.6-11.8 12.7-.3 1-.5 2.1-.3 3 .9 4 2.6 7.7 5.7 10.6 4.2 4 9.5 5.5 15.1 5.8a36.7 36.7 0 0 0 37.6-27.2c.5-2.4.1-3.2-2.1-3.8-8.7-2.1-17.5-3.9-26.6-4.2zm-131.4-42V29.6c0-6.6.1-13.3-.1-19.9-.3-7.3 4-10.7 11.1-8.5 5.8 1.9 11.6 4.1 17.4 6.2l3.6 1.4c6.5 2.3 9.3 7.2 9.7 13.8l.1 3.8v165.5c0 1.9-.1 3.8-.4 5.7-.9 5.1-3.7 8.9-8.4 11-6.6 2.9-13.3 5.4-20 7.9a12 12 0 0 1-7.5.4c-3.4-.9-5.2-3-5.3-6.5v-53.5c-.3-15.9-.3-31.9-.2-47.9z"/></svg>
```

Comment:
Maybe better with 2 text lines? If exist.

Reply:
Should I move half of the SVG to a new line? Will it work?

Reply:
Please show how it may looks (as jpg)
Their YT logo:

<details>
<summary>Details</summary>

![image](https://github.com/AdguardTeam/HostlistsRegistry/assets/8361299/4df13111-3194-4d95-b08e-f1fb0bd0ca49)

</details>